Formula/sftpgo.rb

```diff
@@ -0,0 +1,101 @@
+class Sftpgo < Formula
+  desc "Fully featured SFTP server with optional HTTP/S, FTP/S and WebDAV support"
+  homepage "https://github.com/drakkan/sftpgo"
+  url "https://github.com/drakkan/sftpgo/releases/download/v2.2.3/sftpgo_v2.2.3_src_with_deps.tar.xz"
+  sha256 "6c8676725e86ee3f6ad46a340a84f0da37cab8b6ea7b6aee86b2b96ba5e6671a"
+  license "AGPL-3.0-only"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags", "-s -w", "-trimpath", "-o", "sftpgo"
+    inreplace "sftpgo.json" do |s|
+      s.gsub! "\"users_base_dir\": \"\"", "\"users_base_dir\": \"#{var}/sftpgo/data\""
+      s.gsub! "\"backups_path\": \"backups\"", "\"backups_path\": \"#{var}/sftpgo/backups\""
+      s.gsub! "\"credentials_path\": \"credentials\"", "\"credentials_path\": \"#{var}/sftpgo/credentials\""
+    end
+    bin.install "sftpgo"
+    pkgetc.install "sftpgo.json"
+    pkgshare.install "static", "templates", "openapi"
+    (var/"sftpgo").mkpath
+  end
+
+  def caveats
+    <<~EOS
+      If this is your first installation please open the web administration panel:
+
+      http://localhost:8080/web/admin
+
+      and complete the initial setup.
+
+      The SFTP service is available on port 2022.
+      If the SFTPGo service does not start, make sure that TCP ports 2022 and 8080 are not used by other services
+      or change the SFTPGo configuration to suit your needs.
+
+      SFTPGo service runs as user "#{ENV["USER"]}".
+
+      Default data location:
+
+      #{var}/sftpgo
+
+      Configuration file location:
+
+      #{pkgetc}/sftpgo.json
+
+      Getting started guide:
+
+      https://github.com/drakkan/sftpgo/blob/main/docs/howto/getting-started.md
+
+      Step-to-step tutorials:
+
+      https://github.com/drakkan/sftpgo/tree/main/docs/howto
```

Comment:
```suggestion
      Default data location:

      #{var}/sftpgo

      Configuration file location:

      #{pkgetc}/sftpgo.json
```
Most of this belongs in upstream documentation.